src/convolutions.jl

```diff
@@ -3,7 +3,7 @@ export plan_conv_buffer, plan_conv_psf_buffer
 
 
 """
-    conv(u, v[, dims])
+conv(u, v[, dims])
```

Comment:
please revert all the tabbing changes.

The function has to be indented, the description not!
https://github.com/invenia/BlueStyle#documentation